students/MzKhan/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,79 @@
+'''
+Name: Muhammad Khan
+Date: 03/20/2019
+Assignment06
+'''
+
+import mailroom4 as m
+import pytest as p
+import os
+import sys
+
+
+def test_global_data():
+    assert m.donors_data["Ashley Wiggins"] == [55.66,270,1000]
+    for name, donation in zip(m.donors_name,m.donations):
+        assert m.donors_data[name] == donation
```

Comment:
doesn't look like this is testing any of your code - just testing python dictionary